packages/homepage/src/pages/blog.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { graphql, Link } from 'gatsby';
+import { format } from 'date-fns';
+import TitleAndMetaTags from '../components/TitleAndMetaTags';
+import PageContainer from '../components/PageContainer';
+
+import {
+  Title,
+  PostDate,
+  AuthorImage,
+  Author,
+} from '../components/PostElements';
+
+import {
+  Posts,
+  Subtitle,
+  Thumbnail,
+  Wrapper,
+  Aside,
+  Header,
+  PageTitle,
+  PageSubtitle,
+} from './_blog.elements';
+
+import Layout from '../components/layout';
+import { makeFeed } from '../utils/makePosts';
+
+// UNCOMMENT AT THE BOTTOM IF IT BREAKS
```

Comment:
Will this be uncommented?

Reply:
Only when someone adds a markdown blog post